frontend/src/app/forms/model/steps.model.ts

```diff
@@ -12,7 +12,15 @@ export enum ApplicationStep {
   Submit = 'submit',
 }
 
+export enum hrRatingStep {
+  ApplicantRating = 'applicant-rating',
+  ImpressionInformation = 'impression-information',
+  Finalize = 'finalize',
+}
+
 /**
  * All of the possible steps as an array.
  */
 export const ApplicationStepsArr = Object.values(ApplicationStep);
+
+export const hrRatingStepArr = Object.values(hrRatingStep);
```

Comment:
ya but why create another file. I felt its unnecessary to create another one but will create it if its really required to be separated.

Reply:
The HR steps don't have anything to do with the "main form". I think you should rather think like "what belongs together" instead of "how can I use less files" ;) 

The HR steps are a completely separate thing. They even live in a separate module. And you should usually not include files over the boundaries of modules. (Of course, there are exceptions to this rule...)

Reply:
ok will create a new file.